css/style.css

```diff
@@ -263,3 +265,81 @@ nav a {
   margin-left: 20px;
   color: #253858;
 }
+
+/* Contact form  */
+#contact-form {
```

Comment:
I see that you have made some progress on your web page, but I noticed that you have not made the "Contact Us" section responsive for mobile devices of different screen sizes as I suggested in my previous comment. I would like to remind you that making your web page responsive is a crucial skill for web developers, as it ensures that your web page can adapt to various devices and provide a good user experience. Responsive web design is also important for SEO, as it helps your web page rank higher on search engines.

To fix this issue, I kindly request you to use CSS media queries to adjust the layout and style of the "Contact Us" section according to the screen size of the device. You should work well from a minimum mobile screen size of 375px, and errors in screen sizes less than 375px can be ignored. Furthermore, ensure the mobile version remains responsive until the desktop screen size of 768px.

Here is an example of how you can use media queries to make the "Contact Us" section responsive:

```css
/* Default style for desktop screens */
.contact-us {
  display: flex;
  align-items: center;
  justify-content: space-between;
}

.contact-us form {
    max-width: 400px;
      margin: 20px auto;
}

.contact-us form input, .contact-us form textarea {
        width: 100%;
        margin: 10px 0;
}

textarea {
        max-width: 100%;
        height: 170px;
      }


/* Style for mobile screens */
@media screen and (max-width: 768px) {
 .footer {
    background: url('../images/background/contact\ form\ background\ shapes.svg');
    background-color: #6070ff;
    text-align: center;

    h2,
    p {
      max-width: 500px;
      margin: auto;
      text-align: center;
    }

    form {
      .btn {
        margin: auto;
      }
    }
  }
}
```

I hope this helps you understand how to make your web page responsive for mobile devices. If you have any questions or concerns about this suggestion, please let me know. I appreciate your cooperation and willingness to learn. Thank you for your attention and keep up the good work! 🌟📝🚀


Here's a video for more explanation:

https://www.loom.com/share/a12a3ac46b85462dbaf9c62a9e38a159?sid=850296a2-4db3-4ad5-ace6-5ec9de821e99